pkg/storage/user.go

```diff
@@ -0,0 +1,109 @@
+package storage
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+
+	"go.mongodb.org/mongo-driver/bson/primitive"
+
+	"go.mongodb.org/mongo-driver/bson"
+
+	"github.com/HarlamovBuldog/social-tournament-service/pkg/sts"
+)
+
+func (db *DB) AddUser(ctx context.Context, name string) (string, error) {
+	insertResult, err := db.conn.Collection(usersCollectionName).InsertOne(ctx, sts.User{
+		Name:    name,
+		Balance: 0,
```

Comment:
since balance is a float64 0 will be a default value, so that 0 assignment is not needed